docs/migration/2211_0/2211-typescript-manual.doc.md

```diff
@@ -94,3 +94,9 @@ Constructor has been extended with a new dependency to `ConfiguratorQuantityServ
 ## ActiveCartService
 
 - `WindowRef` is now a required constructor dependency.
+
+# Changes in the core library
+
+## ClientAuthStoreModule
+
+- `HttpClientModule` has been removed from the `imports` property inside `NgModule` metadata. This should not be a problem, as long as you import `HttpClientModule` in your `app.module.ts`.
```

Comment:
HttpClientModule is not anymore in our app.module.ts,
As it has been substitute by:
```
import {
  provideHttpClient,
  withFetch,
  withInterceptorsFromDi,
} from '@angular/common/http';
```
Maybe rephrasing to avoid confusion?eg: import 'httpClientModule' or inject HttpClient via 'provideHttpClient'

Reply:
```suggestion
- `HttpClientModule` has been removed from the `imports` property inside `NgModule` metadata. This should not be a problem, as long as you import `HttpClientModule` in your `app.module.ts`. Alternatively to _importing_ `HttpClientModule` in your `app.module.ts` you can _provide_ `provideHttpClient(withInterceptorsFromDi())` (optionally with `withFetch()`). For more, see https://angular.io/api/common/http/provideHttpClient
```